test/mocks/PSMVariant2Mock.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import { MockERC20 } from "lib/erc20-helpers/src/MockERC20.sol";
+
+// Mock of lite psm: https://github.com/makerdao/dss-lite-psm/blob/374bb08b09a3f4798858fd841bab8e79719266c8/src/DssLitePsm.sol
+contract PSMVariant2Mock {
+
+    MockERC20 public dai;
+    MockERC20 public gem;
+
+    uint256 public tin;
+    uint256 public tout;
+
+    uint256 private to18ConversionFactor;
+
+    constructor(MockERC20 _dai, MockERC20 _gem) {
+        dai = _dai;
+        gem = _gem;
+
+        to18ConversionFactor = 10 ** (18 - gem.decimals());
+    }
+
+    function sellGem(address usr, uint256 gemAmt) external returns (uint256 daiOutWad) {
```

Comment:
Lets update both of these functions to use `pocket` so that the integration tests can use the exact same assertions/approaches

Reply:
good call, added the pocket